### PR TITLE
Merge Release 36 to main

### PR DESCRIPTION
**Pull request by JoeHS**

### CODE REVIEW

File: README.md
Comment (LarryOG) [inline README.md:1]: seems out of date
(I dont think we use these anymore:
* [dfc-providerportal-courses](https://github.com/SkillsFundingAgency/dfc-providerportal-courses)
* [dfc-providerportal-fileprocessor](https://github.com/SkillsFundingAgency/dfc-providerportal-fileprocessor)
* [dfc-providerportal-tribalexporter](https://github.com/SkillsFundingAgency/dfc-providerportal-tribalexporter)
* [dfc-providerportal-ukrlp](https://github.com/SkillsFundingAgency/dfc-providerportal-ukrlp))

File: src/Dfc.CourseDirectory.Api/Dfc.CourseDirectory.Api.csproj
Comment (LarryOG) [inline src/Dfc.CourseDirectory.Api/Dfc.CourseDirectory.Api.csproj:1]: this is wrong

File: src/Dfc.CourseDirectory.Core/DataStore/Sql/TvpHelper.cs
Comment (LarryOG) [inline src/Dfc.CourseDirectory.Core/DataStore/Sql/TvpHelper.cs:1]: look like we dont need a method starting at line 12 her

File: src/Dfc.CourseDirectory.Services/Models/Enums.cs [hunk 1]
```diff
@@ -77,8 +28,7 @@ public enum FundingOptions
 
     /// <summary>
     /// Obsolete,
-    /// use <see cref="Dfc.CourseDirectory.Core.Models.ApprenticeshipStatus"/>
-    /// or <see cref="Dfc.CourseDirectory.Core.Models.CourseStatus"/> instead.
+    /// use <see cref="Dfc.CourseDirectory.Core.Models.CourseStatus"/> instead.
```
Comment (LarryOG) [inline src/Dfc.CourseDirectory.Services/Models/Enums.cs:31]:
```suggestion
    /// use <see cref="Dfc.CourseDirectory.Core.Models.CourseStatus"/> 
```

Comment (JoeHS) [reply src/Dfc.CourseDirectory.Services/Models/Enums.cs:31]: this is commented out, is it not?

**Review by LarryOG — APPROVED** [review]

**Pull request merged**